governance/xc-admin/packages/crank-executor/package.json

```diff
@@ -22,6 +22,7 @@
     "@pythnetwork/client": "^2.9.0",
     "@solana/web3.js": "^1.73.0",
     "@sqds/mesh": "^1.0.6",
-    "ts-node": "^10.9.1"
+    "ts-node": "^10.9.1",
+    "xc-admin-common": "*"
```

Comment:
anything? 🤔 in rust there is a way for local dependency to be versioned. It helps in not missing the updates if the dependency updates, how would it be here? Of course there is the lock file.

Reply:
I think this is the way you do local dependencies in lerna

Reply:
It's importing the local package